apps/src/gamelab/GameLab.js

```diff
@@ -760,7 +762,7 @@ GameLab.prototype.rerunSetupCode = function() {
   }
   Sounds.getSingleton().muteURLs();
   this.gameLabP5.p5.allSprites.removeSprites();
-  if (this.gameLabP5.spritelab) {
```

Comment:
I have a question about whether this swap upholds the intention of the original conditional. I read `this.studioApp_.isUsingBlockly()` as checking a (boolean) setting and `this.gamelabP5.spritelab` as checking whether an object is defined (to avoid a null pointer exception). Although the `this.gamelabp5.spritelab` is defined/not-defined based on the `this.studioApp_.isUsingBlockly()`, replacing `this.gamelabp5.spritelab` with `this.studioApp_.isUsingBlockly()` seems to add a layer of obfuscation to checking whether the `this.gamelab.spritelab` is defined and so that conditional then becomes `this.gamelab.spritelab` should have been defined. I'm probably over-thinking this, but I do wonder this switch maintains the intended logic of these conditionals.

Reply:
I think it's okay, because `gameLabP5.spritelab` is defined if and only if `studioApp_.isUsingBlockly`
If you don't think this is actually cleaner though, I can close this PR. It doesn't really need to happen, I just thought it'd be simpler this way